### ISSUE TITLE
Issue by activating Plugin

### ISSUE THREAD

**Issue by justfor-it** [burnacid/MyBB-Form-Creator#131]
**Describe the bug**

**To Reproduce**
Activate the plugin and you get this message
Have already reinstalled the plugin but still the same
![Screenshot_3](https://user-images.githubusercontent.com/68807794/192995472-2f54b69b-15ae-44d7-aece-c08ff13009f7.png)

**Desktop (please complete the following information):**
 - MyBB version: 	1.8.30
 - Plugin version: 2.6.5
 - PHP version: 7.4.30
 - MySQL version: MySQLi 8.0.30

**Please add an export of your form if the issue is within processing the form**

**Comment by burnacid** [comment]
I've seen this behavior before. Does it activate? This could be a false positive

**Comment by justfor-it** [comment]
Yes it is acrivated but the message will not go away. So that’s the strange thing

**Comment by burnacid** [comment]
You can ignore it for now. It should work. Still need to resolve it.

**Comment by justfor-it** [comment]
Thanks Hero!

**Comment by burnacid** [comment]
Leaving this open as I need to improve something in the future.

**Comment by burnacid** [comment]
Included in v2.6.6

**Comment by burnacid** [comment]
Published v3.0
